css/style.css

```diff
@@ -0,0 +1,173 @@
+* {
+     -moz-box-sizing: border-box; /* Firexfox */
+     -webkit-box-sizing: border-box; /* Safari/Chrome/iOS/Android */
+     box-sizing: border-box; /* IE */
+}
+
+.clearfix:after {
+     content: ".";
+     display: block;
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+body{
+    font-family: 'Helvetica', 'Arial', sans-serif;
+}
+/*impact*/
+
+/*.container{
+  width:970px;
+  margin:auto;
+}*/
+
+.container{
+  min-width:400px;
+  max-width:1200px;
+  margin: auto;
+}
+/*Can set this way and only have on container for the page*/
+
+header{
+  background-color:black;
+  width:100%;
+  height:100px;
+  border-bottom: 3px solid yellow;
+  padding: 10px;
+}
+
+header h2{
+  color: yellow;
+  margin-top: 0px;
+/*  padding: 10px;*/
+}
+
+.nav{
+  width:500px;
+  float:right;
+  text-align: center;
+}
+
+.nav a{
+  color:white;
+  padding: 10px;
+  font-size: 14px;
+  font-family: Impact;
+}
+
+#tagline{
+  width:100%;
+  height:60px;
+  background-color:#A9A9A9;
+  text-align: center;
+}
+
+#tagline h1{
+  font-size: 24px;
+  font-style: italic;
+  margin-top:0px;
+  padding:15px;
+}
+
+.taste{
+  color:#DCDCDC;
+  text-decoration: line-through;
+}
+
+.underline{
+  /*text-decoration: underline;
+  text-decoration-color:#FFFF00;*/
+  border-bottom: 2px #FFFF00 solid;
+}
+
+#main{
+  width: 100%;
+  background-image: url(images/coworking.jpg);
+  background-size:cover;
+  background-position: 60%;
+  height: 380px;
+}
+
+#transparent{
+  height: 320px;
+  width:220px;
+  opacity: 0.8;
+  float:right;
+  background-color:#E4E0E0;
+  margin: 20px 20px 10px 10px;
+  padding:20px;
+  border:#A9A9A9 solid 1px;
+}
+
+#yellow_button{
+  font-weight: bolder;
+  font-size: 12px;
+  border: 5px;
+  background-color: #F9E42E;
+  color; #606161;
+  padding: 5px 15px 5px 15px;
+  box-shadow: 0px 2px grey;
+}
+
+#actions {
+  height: 220px;
+  width: 100%;
+  float:left;
+  margin-bottom: 30px;
+}
+
+h4{
+  font-size: 20px;
+}
+
+.action{
+  height: 220px;
+  width:230px;
+  float:left;
+  text-align:left;
+  margin: 10px;
```

Comment:
instead of using opacity
try using background color using RGBA value

opacity makes all the child blurry as well